### ISSUE TITLE
Traceback before importing finishes

### ISSUE THREAD

**Issue by kargakis** [mimblewimble/grin-explorer#33]
```
Stored block 00e3a7aee38f533af6dc1efb877657c7732c72d9d11194d09d74be18b7315060 @ 7968
  Marked block 00e3a7 as previous of block 00d2e3
Stored block 02acf66db0199bcf5f528475cb6d1687680a6b0f7193183d1ed17e399d5ba48f @ 7967
  Marked block 02acf6 as previous of block 00e3a7
Stored block 00672e00ef1d11970ebbb13fcbdacb333b0b3ff064a16357f7f17a352d74b409 @ 7966
  Marked block 00672e as previous of block 02acf6
Stored block 0070d72cdb2a699ed180672572ef82bf122d08b44ae22e30ed087bacac037fe9 @ 7965
  Marked block 0070d7 as previous of block 00672e
Stored block 00ec07f76524a1c1705668221964989ed49552a120bf878b507d760d7160fc76 @ 7964
  Marked block 00ec07 as previous of block 0070d7
Stored block 0239f4657058df78f70e023b2a46414f31bd346251615938e32f093073337589 @ 7963
  Marked block 0239f4 as previous of block 00ec07
Stored block 01b66fa15c67a1df43264eb27882933cf2cd9243e6a9a67b60a7ea284fb8220d @ 7962
  Marked block 01b66f as previous of block 0239f4
Traceback (most recent call last):
  File "./grinexplorer/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/home/mkargaki/projects/grin-explorer/venv/lib64/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/mkargaki/projects/grin-explorer/venv/lib64/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/mkargaki/projects/grin-explorer/venv/lib64/python3.6/site-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/mkargaki/projects/grin-explorer/venv/lib64/python3.6/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/home/mkargaki/projects/grin-explorer/grinexplorer/blockchain/management/commands/import_from_tip.py", line 47, in handle
    (status, block_hash, prev_hash) = self.fetch_and_store_block(hash, parent)
  File "/home/mkargaki/projects/grin-explorer/grinexplorer/blockchain/management/commands/import_from_tip.py", line 61, in fetch_and_store_block
    block_data = requests.get(self.API_BASE + "blocks/" + hash).json()
  File "/home/mkargaki/projects/grin-explorer/venv/lib64/python3.6/site-packages/requests/models.py", line 892, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib64/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
@hendi

**Comment by kargakis** [comment]
After restarting the process, I get this
```
$ python3 ./grinexplorer/manage.py import_from_tip http://127.0.0.1:13413
height=10919, tip=01eb56a93f0c82bc9938ff7788960ac38613e266d4a171cbe43cd372f6012e5b

Stored block 01eb56a93f0c82bc9938ff7788960ac38613e266d4a171cbe43cd372f6012e5b @ 10919
Stored block 013aa818df9edca4f235c751a02edaf601784520476dc6b39b717078f2a6cd4e @ 10918
  Marked block 013aa8 as previous of block 01eb56
Stored block 003ca230fec9f7c83226c178c8cb27901b54a3375858f9cf8c00f093e7eafe01 @ 10917
  Marked block 003ca2 as previous of block 013aa8
Stored block 02369cb2055ad587884462ed66098ba400bb89191ba7c3c63b5f5a57b9ece87e @ 10916
  Marked block 02369c as previous of block 003ca2
Stored block 011711091da6ea588ed4b757cee63fdd3cd4ced535da30a957c2ea3d07f52f14 @ 10915
  Marked block 011711 as previous of block 02369c
Stored block 01d6ce01291bdab0a02951bd9ed044842b855da9de10f01096a9707778376657 @ 10914
  Marked block 01d6ce as previous of block 011711
Block 02dee96cbcf6edeec5653b12fda07702db26b09a2742a2ae6b1e86dc9aea3806 already exists @ 10913
Stored block 02dee9 as previous of block 01d6ce
== exiting early
```

**Comment by kargakis** [comment]
Also, `runserver` still returns 500s.

**Comment by hendi** [comment]
The 500 error should be fixed by 4839370
